app/services/libanswers_api_service.rb

```diff
@@ -0,0 +1,67 @@
+# frozen_string_literal: true
+
+class LibanswersApiService
+  class LibanswersApiError < StandardError; end
+  attr_reader :work
+
+  def initialize(work_id)
+    @work = Work.find(work_id)
+  end
+
+  def admin_create_curation_ticket
+    conn = Faraday.new(url: host) do |f|
+      f.headers['Authorization'] = "Bearer #{oauth_token}"
+    end
+    response = conn.post(create_ticket_path, "quid=#{scholarsphere_queue_id}&" +
+                                             "pquestion=#{subject}&" +
+                                             "pname=#{send_to_name}&" +
+                                             "pemail=#{send_to_email}")
+    if response.env.status == 200
+      host + JSON.parse(response.env.response_body)['ticketUrl']
+    else
+      raise LibanswersApiError, JSON.parse(response.env.response_body)['error']
+    end
+  rescue Faraday::ConnectionFailed => e
+    raise LibanswersApiError, e.message
+  end
+
+  private
+
+    def send_to_name
+      work.depositor.display_name
+    end
+
+    def send_to_email
+      work.depositor.email
```

Comment:
Refactoring suggestion:  I think it would be better to have the work delegate to its depositor to access `email` and `display_name`.